README.md

```diff
@@ -10,9 +10,8 @@
 * Update `App.config` file with your BrowserStack Username and Access Key(https://www.browserstack.com/accounts/settings)
 
 ## Running your tests from Test Explorer via NUnit Test Adapter
-- To run a single test, run test with fixture `single`
```

Comment:
Keep run single test below . Order should be parallel > local > single

Reply:
But in other repos, we have removed this. Should we keep it here?